src/sysupdate/sysupdate.c

```diff
@@ -1576,7 +1577,17 @@ static int verb_components(int argc, char **argv, void *userdata) {
                                 break;
                         }
 
-                        if (de->d_type != DT_DIR)
+                        if (de->d_type == DT_LNK) {
+                                if (fstatat(dirfd(d), de->d_name, &st, AT_NO_AUTOMOUNT) < 0) {
```

Comment:
needs to take arg_root into account, i.e. symlinks must be followed relative to that...